### DISCUSSION TITLE
Why is `public func application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask` not `open`?

### DISCUSSION THREAD

**Discussion by abulenok** [expo/expo#31383] [Archived: Questions] [answered]
link to the method: https://github.com/expo/expo/blame/main/packages/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift#L304

**Comment by abulenok** [comment]
all other `aplication` methods are `open`

**Comment by tsapeta** [comment, marked as answer]
Yes, it should be `open`. Could you open a PR to fix it?

**Reply by abulenok** [reply]
Sure, https://github.com/expo/expo/pull/31398
